### DISCUSSION TITLE
Icon explanation

### DISCUSSION THREAD

**Discussion by akrogun** [fast4x/RiMusic#473] [Q&A]
Hi, I need some explanations about icon on Favorite Playlist. What are the icons mean?

Thanks in advance!

![IMG_20240122_103348](https://github.com/fast4x/RiMusic/assets/34392285/a8896bf0-5547-4fca-a005-28d1b38a7b4f)

**Comment by fast4x** [comment]
Hi, from top left:

- download all
- remove all downloaded
- enqueue
- shuffle
- lock/unlock ordering
- menu playlist

From bottom left
- search in playlist 
- order by
-- by recent played
-- by position
-- by most played
-- by title
-- by artist
-- sort order asc/desc for all order option

**Reply by fast4x** [reply]
AC/DC i love it 😂

**Reply by ikanakova** [reply]
I put an explanation of the icons in the FAQ: https://github.com/fast4x/RiMusic/wiki/FAQs#-icon-explanation
I noticed that two icons are new, so I'll create them and add them to assets and I'll add it to the FAQ.

**Reply by akrogun** [reply]
just curious.... order by position, what position ? And one more thing, if I re-order using unlocking order icon, why the list won't stick to what I want?

**Comment by fast4x** [comment]
The songs in all playlist are numered from 1 to x... 
"if I re-order using unlocking order icon, why the list won't stick to what I want?" there is a bug...

**Reply by ikanakova** [reply]
> there is a bug...

see #251

**Reply by ikanakova** [reply]
> The songs in all playlist are numered from 1 to x...

Wouldn't an icon like this be better?

<img src="https://raw.githubusercontent.com/ikanakova/RiMusic/icons/assets/icons/position.svg" width="20">

I made it like this one:

<img src="https://raw.githubusercontent.com/fast4x/RiMusic/master/assets/ionicons/text.svg" width="20">

**Reply by fast4x** [reply]
I'll update it

**Reply by ikanakova** [reply]
It is in pull request #474
